xpcPlugin/DataManager.h

```diff
@@ -410,6 +410,13 @@ namespace XPC
 		/// \param value The flaps settings. Should be between 0.0 (no flaps) and 1.0 (full flaps).
 		static void SetFlaps(float value);
 
+        /// Executes a command
+        ///
+        /// \param comm   The name of the command to execute.
+        static void Execute(const std::string& comm);
+
+        /// Gets a default value that indicates that a dataref should not be changed.
```

Comment:
I think this was included accidentally?

Also, the indentation looks off here. That usually means there's mixed tabs and spaces.

Reply:
That's a weird one. I'll look into that.

Reply:
Actually this was in the original pull request: https://github.com/nasa/XPlaneConnect/pull/120/commits/9bfa8c3f24eb1156883f1fd19e5f6c24f7e632c8#diff-c14463a65b9022a134315f9ddc438ec4